patches/minecraft/net/minecraft/entity/player/EntityPlayer.java.patch

```diff
@@ -242,15 +242,29 @@
          if (p_71059_1_.func_70075_an())
          {
              if (!p_71059_1_.func_85031_j(this))
-@@ -1340,6 +1413,7 @@
+@@ -1180,9 +1253,11 @@
+                     boolean flag2 = flag && this.field_70143_R > 0.0F && !this.field_70122_E && !this.func_70617_f_() && !this.func_70090_H() && !this.func_70644_a(MobEffects.field_76440_q) && !this.func_184218_aH() && p_71059_1_ instanceof EntityLivingBase;
+                     flag2 = flag2 && !this.func_70051_ag();
+ 
+-                    if (flag2)
++                    net.minecraftforge.event.entity.player.CriticalHitEvent hitResult = net.minecraftforge.common.ForgeHooks.getCriticalHit(this,p_71059_1_,flag2, 1.5F);
++                    flag2 = hitResult!=null;
++                    if(flag2)
```

Comment:
This line is an unnecessary change, it should be `if (flag2)` with a space, to match the line that was there before.